x-pack/plugins/ml/public/application/settings/calendars/edit/new_calendar.js

```diff
@@ -328,6 +330,8 @@ class NewCalendarUI extends Component {
       isGlobalCalendar,
     } = this.state;
 
+    const helpLink = getDocLinks().links.ml.guide;
```

Comment:
Is it worth linking to the Calendars [page](https://www.elastic.co/guide/en/machine-learning/master/ml-calendars.html) here rather than the top level of the ML guide?  If so, looks like you'd need to add that calendars URL to the doc links service.

Reply:
Yup, that makes sense!  I've made those changes.